shared/rakefiles/xk_config.rake

```diff
@@ -15,19 +16,30 @@ end
 # (and a new Serviceaccount Key) whenever @gcp_creds_file changes. This is not
 # what we want, i.e. don't create a new Key/key file when @gcp_creds_file
 # changes because of :configure_kubectl.
-@serviceaccount_key_file = ENV["TF_VAR_serviceaccount_key"]
-task :configure_serviceaccount => [@gcp_creds_file, :configure_current_project] do
+@serviceaccount_key_file = "/project/live/#{ENV['ENV']}/secrets/kube-system/owner.json"
+task :configure_serviceaccount => [@gcp_creds_file] do
   # TODO: This command is duplicated from exekube's gcp-project-init (and
   # hardcodes 'projectowner' instead of $SA_NAME which is only defined in
   # gcp-project-init). If gcp-project-init becomes idempotent (GPII-2989,
   # https://github.com/exekube/exekube/issues/92), or if this 'keys create'
   # step moves somewhere else in exekube, call this command from that place
   # instead.
-  sh "
-    [ -f $TF_VAR_serviceaccount_key ] || \
-    gcloud iam service-accounts keys create $TF_VAR_serviceaccount_key \
-      --iam-account projectowner@$TF_VAR_project_id.iam.gserviceaccount.com
-  "
+  unless File.file?(@serviceaccount_key_file)
+    sh "
+      gcloud iam service-accounts keys create #{@serviceaccount_key_file} \
+        --iam-account projectowner@$TF_VAR_project_id.iam.gserviceaccount.com
+    "
+  end
+end
+
+@default_creds_file = "/root/.config/gcloud/application_default_credentials.json"
```

Comment:
I find this name confusing. How about `@application_default_credentials_file`, `@application_default_creds_file`, `@app_default_creds_file`, or `@adc_file`?

Reply:
+1, `@application_default_credentials_file`

Reply:
I choose `@app_default_creds_file` to be match style of `@gcp_creds_file`.